tests/_common.py

```diff
@@ -1,6 +1,8 @@
 import os
```

Comment:
If we use `Path()` we can replace the `os.path` calls. I can add this.

Reply:
and yes please! I'll merge this tomorrow morning now